api/filter/rate_limit.proto

```diff
@@ -2,6 +2,19 @@ syntax = "proto3";
 
 package envoy.api.v2.filter;
 
+import "google/protobuf/duration.proto";
+
+// TODO(htuch): This should be the shared canonical RateLimitDescriptor
+// when we import the rate limit protos
+// (https://github.com/lyft/envoy-api/issues/26).
+message RateLimitDescriptor {
+  message Entry {
+    string key = 1;
+    string value = 2;
+  }
+  repeated Entry entries = 1;
```

Comment:
why not a map from string to string?

Reply:
I think its because there is only one key/value pair. This was already defined by existing rate limit proto. Just reusing it. So, its a question for @htuch and co

Reply:
SGTM, I don't think the rate limit proto is used anywhere.

Reply:
Do you want to update this one?